### PR TITLE
Variation cleanup, improved debug colors

### PR DESCRIPTION
**Pull request by dimgrav**
Looks like Pop Light is basically Pop Color, but with a few malconfigured colors, as seen below:

Screenshot from 2020-09-28 20-59-00

I merged Pop Light with Pop Color and left two valid variations (light/dark). Also provided some consistent debug color styles for both variations (issue #8 for reference).

### CODE REVIEW

File: poptheme/themes/Pop-Light-color-theme.json [hunk 1]
```diff
@@ -1,207 +1,207 @@
...
-         "badge.background": "#F15D22",
-         "progressBar.background": "#48B9C7",
+		"badge.background": "#F15D22",
+		"badge.foreground": "#FFF",
+		"progressBar.background": "#48B9C7",
 
-         "dropdown.background": "#F6F6F6",
-         "dropdown.foreground": "#403C3A",
-         "dropdown.border": "#63524b00",
+		"dropdown.background": "#F6F6F6",
```
Comment (subpop) [inline poptheme/themes/Pop-Light-color-theme.json]: Looks like you've got some tab vs. space issues here. I'd suggest cleaning these up to make a smaller diff that contains *just* the changes you're proposing in this PR.

Comment (dimgrav) [reply poptheme/themes/Pop-Light-color-theme.json]: Ah, didn't notice that, thanks for pointing it out!

**Comment by ArtisanByteCrafter** [comment]
Thanks for the contributions! I haven't had much time to work on this lately, but I'll be sure to review soon and get it updated once we get the issues raised all sorted out.

File: poptheme/themes/Pop-Light-color-theme.json (hunk 1 above)
Comment (ArtisanByteCrafter) [reply poptheme/themes/Pop-Light-color-theme.json]: Agreed, once PR is only proposed changes, i'm happy to review and merge.

**Review by ArtisanByteCrafter — CHANGES_REQUESTED** [review]
The changes i think you meant look good to me. If you could sort the indent issues in the diff, it would be easier to review, then I'll be happy to merge this PR. Thanks!

**Comment by dimgrav** [comment]
@ArtisanByteCrafter will do the necessary cleanup asap, unfortunately I didn't have time this week!

**Comment by dimgrav** [comment]
@ArtisanByteCrafter  I deleted the original `Pop-Light-color-theme.json` and renamed `Pop-color-theme.json` -> `Pop-Light-color-theme.json`, then re-applied the specific changes per line.

**Comment by subpop** [comment]
@dimgrav take a look a the resulting patch set produced by the PR https://github.com/ArtisanByteCrafter/VSCodePopTheme/pull/11/files. It still contains changes to whitespace only. Is this a tabs vs. spaces issue perhaps?

**Comment by dimgrav** [comment]
> @dimgrav take a look a the resulting patch set produced by the PR https://github.com/ArtisanByteCrafter/VSCodePopTheme/pull/11/files. It still contains changes to whitespace only. Is this a tabs vs. spaces issue perhaps?

The whitespace diff is due to renaming `Pop-color-theme.json` -> `Pop-Light-color-theme.json`. The original `Pop-Light-color-theme.json` had the same indentation as `Pop-Dark-color-theme.json`. There was no consistent indentation between  the theme .json files.

**Comment by subpop** [comment]
Ahhh yea, that makes sense.

**Comment by ArtisanByteCrafter** [comment]
Version 1.0.4 has been published to the VSCode Marketplace to resolve these issues. Thanks again!

**Pull request merged**